include/directx/d3dx12.h

```diff
@@ -2101,8 +2101,8 @@ inline UINT64 UpdateSubresources(
     {
         for (UINT i = 0; i < NumSubresources; ++i)
         {
-            CD3DX12_TEXTURE_COPY_LOCATION Dst(pDestinationResource, i + FirstSubresource);
-            CD3DX12_TEXTURE_COPY_LOCATION Src(pIntermediate, pLayouts[i]);
+            const CD3DX12_TEXTURE_COPY_LOCATION Dst(pDestinationResource, i + FirstSubresource);

```

Comment:
Also a few places that can be marked const per [Con. 4](https://devblogs.microsoft.com/cppblog/check-for-const-correctness-with-the-c-core-guidelines-checker).